conf/broker.conf

```diff
@@ -1481,4 +1481,17 @@ tlsEnabled=false
 
 # Enable Key_Shared subscription (default is enabled)
 # @deprecated since 2.8.0 subscriptionTypesEnabled is preferred over subscriptionKeySharedEnable.
-subscriptionKeySharedEnable=true
\ No newline at end of file
+subscriptionKeySharedEnable=true
+
+# Max number of "acknowledgment holes" that can be stored in Zookeeper. If number of unack message range is higher
+# than this limit then broker will persist unacked ranges into bookkeeper to avoid additional data overhead into
+# zookeeper.
+# Deprecated: use managedLedgerMaxUnackedRangesToPersistInMetadataStore
+managedLedgerMaxUnackedRangesToPersistInZooKeeper=-1
+
+# If enabled, the maximum "acknowledgment holes" will not be limited and "acknowledgment holes" are stored in
+# multiple entries.
+persistentUnackedRangesWithMultipleEntriesEnabled=false
```

Comment:
They are introduced by #15099 and #15607. 
We don't need to cherry-pick them to branch-2.10, right?

Reply:
Yes, it is useless.